src/oncall/ui/static/js/oncall.js

```diff
@@ -3484,6 +3484,19 @@ var oncall = {
         });
       });
     }
+  },
+  sanitize: function (string) {
```

Comment:
Why not use the Handlebars built-in escape function (afaik Oncall still uses handlebars)? Or just use Handlebars formatting for these strings?

Reply:
Good suggestion, Handlebars.escapeExpression(string) does the exact same thing as this sanitize function so I might as well use that. https://handlebarsjs.com/api-reference/utilities.html#handlebars-escapeexpression-string